Complete DuckDNS add-on documentation (#749)
home-assistant/hassio-addons: duckdns/README.md

## How to use

1. Visit [DuckDNS.org](https://www.duckdns.org/) and create an account by logging in through any of the available account services (Google, Github, Twitter, Persona).

> 2. In the `Domains` section, type the name of the sub domain you wish to register and click `add domain`.

Reviewer: ```suggestion
2. In the `Domains` section, type the name of the subdomain you wish to register and click `add domain`.
```

> 3. If registration was a success, the sub domain is listed in the `Domains` section along with `current ip` being the public IP address of the device you are currently using to access duckdns.org. The IP address will be updated by the DuckDNS add-on.

Reviewer: ```suggestion
3. If registration was a success, the subdomain is listed in the `Domains` section along with `current ip` being the public IP address of the device you are currently using to access `duckdns.org`. The IP address will be updated by the DuckDNS add-on.
```

4. In the DuckDNS add-on configuration, perform the following:
    - Copy the DuckDNS token (listed at the top of the page where account details are displayed) from `duckdns.org` and paste into the `token` option.

>     - Update the `domains` option with the full domain name you registered. E.g. "my-domain.duckdns.org".

Reviewer: ```suggestion
    - Update the `domains` option with the full domain name you registered. E.g., `my-domain.duckdns.org`.
```

}
```

### Option group `lets_encrypt`

> ---

Reviewer: Let us remove the line separator, there is no begin/end of this section.

The following options are for the option group: `lets_encrypt`. These settings
only apply to Let's Encrypt SSL  certificates.

> #### Option `lets_encrypt`: `accept_terms`

Reviewer: Let us make it consistent with how we handle services on our documentation website.

```suggestion
### Option `lets_encrypt.accept_terms`
```

Once you have read and accepted the Let's Encrypt [Subscriber Agreement](https://letsencrypt.org/repository/), change value to `true` in order to use Let's Encrypt services.

> #### Option `lets_encrypt`: `certfile`

Reviewer: ```suggestion
### Option `lets_encrypt.certfile`
```

The name of the certificate file generated by Let's Encrypt. The file is used for SSL by Hass.io add-ons and is recommended to keep the filename as-is (`fullchain.pem`) for compatibility.

**Note**: _The file is stored in `/ssl/`, which is the default for Hass.io_

> #### Option `lets_encrypt`: `keyfile`

Reviewer: ```suggestion
#### Option `lets_encrypt.keyfile`
```

The name of the private key file generated by Let's Encrypt. The private key file is used for SSL by Hass.io add-ons and is recommended to keep the filename as-is (`privkey.pem`) for compatibility.

**Note**: _The file is stored in `/ssl/`, which is the default for Hass.io_

### Option: `token`

The DuckDNS authentication token found at the top of the DuckDNS account landing page. The token is required to make any changes to the subdomains registered to your account.

### Option: `domains`

> A list of DuckDNS domains registered under your account. An acceptable naming convention is "my-domain.duckdns.org".

Reviewer: ```suggestion
A list of DuckDNS subdomains registered under your account. An acceptable naming convention is `my-domain.duckdns.org`.
```

### Option: `seconds`

The number of seconds to wait before updating DuckDNS subdomains and renewing Let's Encrypt certificates.

## Known issues and limitations

- To log in, DuckDNS requires a free account from any of the following services: Google, Github, Twitter, or Persona.

> - A free DuckDNS account is limited to five sub domains.

Reviewer: ```suggestion
- A free DuckDNS account is limited to five subdomains.
```

> 

Reviewer: Unneeded newline

```suggestion
```
